cmd/main.go

```diff
@@ -100,6 +103,10 @@ func main() {
 		"The minimal timeout between calls to the DNS Provider"+
 			"Controls if we commit to the full reconcile loop")
 	flag.Var(&providers, "provider", "DNS Provider(s) to enable. Can be passed multiple times e.g. --provider aws --provider google, or as a comma separated list e.g. --provider aws,gcp")
+
+	flag.StringVar(&clusterSecretNamespace, "cluster-secret-namespace", "dns-operator-system", "The Namespace to look for cluster secrets.")
```

Comment:
when installed via Kuadrant we will need to set this via to be kuadrant-system. Is it possible to use the downward api to set this as an env var to the namespace we are installed in by default?
https://kubernetes.io/docs/concepts/workloads/pods/downward-api/#available-fields

Reply:
Possibly, i was wondering about this a bit. Originally we said make the default "kuadrant-sytem", but that doesn't really make sense as a default in this repo.

If we can default to using the current namespace when running inside a pod that would be ideal, but we would still need to allow it be set manually, or overridden.

Reply:
Updated https://github.com/Kuadrant/dns-operator/pull/489/commits/a38518245290c4780c001748f1ab7a1c4c0163c5, works well, should now automatically set that correctly when installed via kuadrant operator.